### PR TITLE
PlantDetails now only shows data for plant and dialog shows for specific plant

### LINKED ISSUE #166: MyPlant details only

### PR DESCRIPTION
**Pull request by Peterkmoss**
Closes #166

### CODE REVIEW

File: app/src/main/java/dk/mifu/pmos/vegetablegardening/helpers/Formatter.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package dk.mifu.pmos.vegetablegardening.helpers
+
+import android.content.Context
+import dk.mifu.pmos.vegetablegardening.R
+import java.text.SimpleDateFormat
+import java.util.*
+
+class Formatter(private val context: Context) {
```
Comment (miewetkefust) [inline app/src/main/java/dk/mifu/pmos/vegetablegardening/helpers/Formatter.kt:8]: Do we also use this in lexicon with the dates there?

File: app/src/main/res/layout/fragment_plant_details.xml [hunk 2]
```diff
@@ -24,26 +24,4 @@
         android:id="@+id/gridlayout">
```
Comment (miewetkefust) [inline app/src/main/res/layout/fragment_plant_details.xml:24]: Shouldn't this file be completely deleted? There should be no PlantDetailsFragment right?

Comment (miewetkefust) [reply app/src/main/res/layout/fragment_plant_details.xml:24]: I also can't see that the fragment is deleted

**Review by miewetkefust — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_plant_details.xml (hunk 2 above)
Comment (Peterkmoss) [reply app/src/main/res/layout/fragment_plant_details.xml:24]: No it should be there, it just should not show the details for the MyPlant part - we are still using it for the lexicon

**Review by miewetkefust — APPROVED** [review]

**Pull request merged**